scripts/generator/graphql_generator/csharp/SDK/DefaultQueries.cs.erb

```diff
@@ -8,7 +8,7 @@ namespace Shopify.Unity.SDK {
     /// </summary>
     public class DefaultQueries {
         public static readonly int MaxPageSize = 250;
-        public static readonly int MaxProductPageSize = 20;
+        public static readonly int MaxProductPageSize = 10;
```

Comment:
Is this change intended? Just making sure since it wasn't mentioned in the PR description.

Reply:
Yes it's intended. I think 10 gives us room to grow while still not being a major hit for users. These integration tests really informed those numbers.